src/modules/Sharpen/Module.js

```diff
@@ -0,0 +1,46 @@
+/*
+  Sharpen an image
+*/
+module.exports = function Sharpen(options, UI) {
+
+  var defaults = require('./../../util/getDefaults.js')(require('./info.json'));
+  options.sharpen = options.sharpen || defaults.sharpen;
```

Comment:
can this option be named something more descriptive like `sharpenStrength`?

Reply:
Sure thing!